python/src/trezorlib/btc.py

```diff
@@ -175,7 +177,13 @@ def sign_message(
     )
 
 
-def verify_message(client, coin_name, address, signature, message):
+def verify_message(
+    client: "TrezorClient",
+    coin_name: str,
+    address: str,
+    signature: bytes,
+    message: Union[str, bytes],
```

Comment:
`Union[str, bytes]` can be imported `from typing import AnyStr`

Reply:
dtto for all usages of `Union[str, bytes]` elsewhere

Reply:
Done in 4cfe54a